plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/KotlinJsPluginTests.kt

```diff
@@ -0,0 +1,73 @@
+package org.jlleitschuh.gradle.ktlint
```

Comment:
Thanks for staying on top of these compatibility tests and keeping them refactored as they need to be changed.